fix(docs): update how to add/enable a new language (#48774)
freeCodeCamp/freeCodeCamp: docs/how-to-enable-new-languages.md

      SuperBlocks.RelationalDb,
      SuperBlocks.BackEndDevApis,
      SuperBlocks.QualityAssurance,
      SuperBlocks.SciCompPy,
      SuperBlocks.DataAnalysisPy,
      SuperBlocks.InfoSec,
      SuperBlocks.MachineLearningPy
    ],
    [CurriculumMaps.Learn]: {
      [TranslationStates.Audited]: {
        [SuperBlockStates.Current]: [
          SuperBlocks.RespWebDesignNew,
          SuperBlocks.JsAlgoDataStruct,
          SuperBlocks.FrontEndDevLibs,
          SuperBlocks.DataVis,
          SuperBlocks.RelationalDb,
          SuperBlocks.BackEndDevApis,
          SuperBlocks.QualityAssurance,
          SuperBlocks.SciCompPy,
          SuperBlocks.DataAnalysisPy,
          SuperBlocks.InfoSec,
          SuperBlocks.MachineLearningPy,
          SuperBlocks.CodingInterviewPrep
        ],
        [SuperBlockStates.New]: [],
        [SuperBlockStates.Upcoming]: [SuperBlocks.JsAlgoDataStructNew],
        [SuperBlockStates.Legacy]: [SuperBlocks.RespWebDesign]
      },
      [TranslationStates.NotAudited]: {
        [SuperBlockStates.Current]: [],
        [SuperBlockStates.New]: [],
        [SuperBlockStates.Upcoming]: [],
        [SuperBlockStates.Legacy]: []
      }
    }
  }
}
```

> The order of the superblocks in this object is how they appear on the "Landing" page and "Learn" maps. Follow the comments in that file so you know how you are allowed to order the superblocks, then move them to their proper places for the new language.

Reviewer: ```suggestion
The order of the superblocks in this object is how they appear on the "Landing" page and "Learn" maps. Follow the comments in that file to know how you can order the superblocks, then move them to their proper places for the new language.
```

---

It feels weird for me to tell someone who is reading the docs, "follow the comments in the file" while they are already in the docs.

```suggestion
The order of the superblocks in this object is how they appear on the "Landing" page and "Learn" maps. Move the superblocks in the new language's arrays, which positions them in their proper places.
```

> [!ATTENTION]
> Do not change the order of any of the keys in the object, just move the superblocks to the different arrays

> The `CurriculumMaps.Landing` array should contain exactly one superblock for all our current certifications; and the `CurriculumMaps.Learn` object should have all existing superblocks in it. Translated superblocks go in `TranslationStates.Audited` and non-translated superblocks go in `TranslationStates.NotAudited`. Each of those two objects has four different states a superblock can be in.

Reviewer: ```suggestion
The `CurriculumMaps.Landing` array should contain exactly one superblock for all our current certifications, and the `CurriculumMaps.Learn` object should have all existing superblocks in it. Translated superblocks go in `TranslationStates.Audited` and non-translated superblocks go in `TranslationStates.NotAudited`. Each of those two objects has four different states a superblock can be in.
```

> - `SuperBlockStates.Current`: means that the superblock is current. e.g. `(New) Responsive Web Design`

Reviewer: silly question, we don't apply the writing style for the challenges in the docs, too. 

Whelp, here is a suggestion if we do apply the writing style.

```suggestion
- `SuperBlockStates.Current`: means that the superblock is current, example: `(New) Responsive Web Design`.
```
